CMakeLists.txt

```diff
@@ -222,6 +222,24 @@ if(APPLE_IOS)
 	set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_ALLOWED_FOR_APPS YES)
 	set(CMAKE_XCODE_ATTRIBUTE_PRODUCT_BUNDLE_IDENTIFIER "${BUNDLE_IDENTIFIER_PREFIX}.$(PRODUCT_NAME)")
 	set(CMAKE_XCODE_ATTRIBUTE_TARGETED_DEVICE_FAMILY "1,2")
+
+	if(CMAKE_CXX_COMPILER_LAUNCHER)
```

Comment:
but what's the point of using CMAKE_CXX_COMPILER_LAUNCHER then? I think we need a dedicated variable for xcode / msvc

Reply:
I wanted to re-use the same variable that works on Linux.

Maybe `option(ENABLE_CCACHE_IOS_MSVC "Enable ccache for iOS or using MSVC" OFF)` ?

Reply:
Since the special code is about ccache, then the option should also reflect it.

Let's do something like this:

```cmake
if(CMAKE_GENERATOR STREQUAL Xcode OR MSVC)
  option(ENABLE_CCACHE OFF)
endif()
```

Just please check if syntax is right, typed from memory and phone